ra_aid/webui/server.py

```
@@ -1,29 +1,37 @@
-from fastapi import FastAPI, Request, WebSocket, WebSocketDisconnect
-from fastapi.middleware.cors import CORSMiddleware
-from fastapi.responses import HTMLResponse
-from fastapi.staticfiles import StaticFiles
-from fastapi.templating import Jinja2Templates
+import asyncio
+from typing import List
+import json
+import threading
+import queue
+import traceback
+import shutil
```

Ruff (0.8.2)

`shutil` imported but unused

Remove unused import: `shutil`

(F401)